views/Login/Layout.tsx

```diff
@@ -6,7 +6,8 @@ import styles from './Login.module.scss'
 import LoginForm from './LoginForm'
 
 interface LoginFormProps {
-  onSubmit: (formData: LoginInput) => void
+  // eslint-disable-next-line no-unused-vars
+  onSubmit: (_formData: LoginInput) => void
```

Comment:
Yes, it have this error 😢 
![image](https://user-images.githubusercontent.com/36519478/172526379-6fbafc22-668a-4be6-8149-ca4cecce5fd4.png)

I tried to avoid this error in eslint conf and updated so is not reporting.